spyder_kernels/customize/spydercustomize.py

```diff
@@ -313,6 +313,15 @@ def __init__(self, completekey='tab', stdin=None, stdout=None,
         self._pdb_breaking = False
 
     # --- Methods overriden by us
+    def set_continue(self):
+        """Stop only at breakpoints or when finished.
```

Comment:
When you're writing multi-line strings, our custom is to leave the initial triple quotes empty. In other words, please rewrite this as

```python
"""
Stop only at breakpoints or when finished.

...
"""
```